---
title: publications
date: 2020-03-07 23:04:17
---
## Papers
<!-- * **W. Ouyang**, Emma Lundberg, ImageJ.JS: ImageJ running in the browser, https://ij.imjoy.io/ (2021) -->
<!-- * J. Bai<sup>*</sup>, **W. Ouyang<sup>*#</sup>**, M. Singh, C. Leterrier, P. Barthelemy, S. Holden, T. Klein, M. Sauer, P. Kanchanawong, N. Bourg, M. Cohen, B. Lelandais, C. Zimmer<sup>#</sup>, ShareLoc – an open platform for sharing localization microscopy data (2021) -->
<!-- * M.R. King, K.M. Ruff, X. Zeng, A.E. Posey, W. Ouyang, E. Lundberg, R.V. Pappu. Nucleolar cores are scaffolded by proteins with lysine and acid-rich disordered regions that drive phase separation. In preparation. -->
* E. Gómez-de-Mariscal, C. García-López-de-Haro, **W. Ouyang**, L. Donati, E. Lundberg, M. Unser, A. Muñoz-Barrutia, D. Sage. *DeepImageJ: A user-friendly environment to run deep learning models in ImageJ, Nature Methods* (2021, in press), [preprint](https://www.biorxiv.org/content/10.1101/799270v3)
* A. Imbert, **W. Ouyang**,  A. Safieddine, E. Coleno, C. Zimmer, E. Bertrand, T. Walter, F. Mueller. *FISH-quant v2: a scalable and modular analysis tool for smFISH image analysis*, Biorxiv (2021, submitted to Nucleic Acids Research). https://doi.org/10.1101/2021.07.20.453024
* X. Hao, J.J. Parmar, B. Lelandais, A. Aristov, **W. Ouyang**, C. Weber & C. Zimmer. *Super-resolution visualization and modeling of human chromosomal regions reveals cohesin-dependent loop structures*. Genome Biology (2021). https://doi.org/10.1186/s13059-021-02343-w
* **W. Ouyang**, R. Bowman, H. Wang, K. E Bumke, J. T Collins, O. Spjuth, J. Carreras-Puigvert, B. Diederich, *An Open-Source Modular Framework for Automated Pipetting and Imaging Applications*, Biorxiv (2021, submitted to Advanced Biology).https://doi.org/10.1101/2021.06.24.449732
* H. Pinkard, N. Stuurman, I.E. Ivanov, **W. Ouyang**, B. Li, et al., *Pycro-Manager: open-source software for customized and reproducible microscope control*. Nature Methods (2021). https://doi.org/10.1038/s41592-021-01087-6, [full text](https://rdcu.be/cghwk)
* **W. Ouyang**, T. Le, H. Xu and E. Lundberg. *Interactive biomedical segmentation tool powered by deep learning and ImJoy*. F1000Research (2021). https://doi.org/10.12688/f1000research.50798.1
* Y. Qin, C. F. Winsnes, E. L. Huttlin, F. Zheng, **W. Ouyang**, J. Park, A. Pitea, J. F. Kreisberg, S. P. Gygi, J. W. Harper, J. Ma, E. Lundberg, T. Ideker, *Mapping cell structure across scales by fusing protein images and interactions*, Nature (2021, in press). [preprint](https://doi.org/10.1101/2020.06.21.163709)
* L. Stenström, D. Mahdessian, C. Gnann, A. J Cesnik, **W. Ouyang**, M. D. Leonetti, M. Uhlén, S. Cuylen‐Haering, P. J Thul, E. Lundberg, *Mapping the nucleolar proteome reveals a spatiotemporal organization related to intrinsic protein disorder*. Molecular Systems Biology (2020). https://doi.org/10.15252/msb.20209469
* **W. Ouyang**, F. Mueller, M. Hjelmare, E. Lundberg, C. Zimmer, *ImJoy: an open-source computational platform for the deep learning era*, Nature Methods, 2019, https://doi.org/10.1038/s41592-019-0627-0, [full text](https://rdcu.be/bYbGO)
* **W. Ouyang**, C. Winsnes, M. Hjelmare, A. Cesnik, L. Åkesson, H. Xu et. al, *Analysis of the Human Protein Atlas Image Classification competition*, Nature Methods, 2019, https://doi.org/10.1038/s41592-019-0658-6, [full text](https://www.nature.com/articles/s41592-019-0658-6)
* **W. Ouyang**, A. Aristov, M. Lelek, X. Hao, C. Zimmer*, *Deep learning massively accelerates super-resolution localization microscopy, Nature Biotechnology*, 2018, https://doi.org/10.1038/nbt.4106, [full text](https://rdcu.be/LGtc), [cover design](https://www.nature.com/nbt/volumes/36/issues/5)
* A. Samacoits, R. Chouaib, A. Safieddine, A. Traboulsi, **W. Ouyang**, C. Zimmer, M. Peter, E. Bertrand, T. Walter, F. Mueller, *A computational framework to study sub-cellular mRNA localization*, Nature Communications, 2018, https://doi.org/10.1038/s41467-018-06868-w

* G. Yang, M. Jiang, **W. Ouyang**, G. Ji, H. Xie, A. M. Rahmani, P. Liljeberg, H. Tenhunen, *IoT-based Remote Pain Monitoring System: from Device to Cloud Platform*, IEEE Journal of Biomedical and Health Informatics, 2017, https://doi.org/10.1109/JBHI.2017.2776351

## Review
 * **W. Ouyang**, C. Zimmer*, *The imaging tsunami: computational opportunities and challenges*, Current Opinion in Systems Biology, 2017, https://doi.org/10.1016/j.coisb.2017.07.011

## News and Highlights
 * [AI is changing the future of life science – but how?](https://www.scilifelab.se/news/ai-is-changing-the-future-of-life-science-but-how/), SciLifeLab(2021, interview)
 * [Sharper signals: how machine learning is cleaning up microscopy images](https://www.nature.com/articles/d41586-021-00023-0), Nature (2021, technology features)
 * [Deep learning takes on tumours](https://www.nature.com/articles/d41586-020-01128-8), Nature (2020, technology features)
 * [ImJoy: a new computational platform for deep learning](https://www.scilifelab.se/news/imjoy-a-new-computational-platform-for-deep-learning/), SciLifeLab(2020, news)
 * [Artificial Intelligence: A New Platform For The Biomedical Community](https://www.pasteur.fr/en/research-journal/news/artificial-intelligence-new-platform-biomedical-community), Institut Pasteur (2020, news)
 * [Deep learning advances super-resolution imaging](https://www.nature.com/articles/s41592-018-0028-9), Nature Methods (2018, Research Highlight)
 * [Super resolution microscopy gets a dose of deep learning](https://physicsworld.com/a/super-resolution-microscopy-gets-a-dose-of-deep-learning/), Physics World (2018)
 * [Aplicando profundamente a aprendizagem à microscopia da localização](https://www.news-medical.net/life-sciences/Applying-Deep-Learning-to-Localization-Microscopy-(Portuguese).aspx), Medical News (2020)
 * [Artificial Intelligence Speeds Up High-resolution Microscopy](https://www.inception-program.fr/en/news/artificial-intelligence-speeds-high-resolution-microscopy), Institut Pasteur (2018, news)
 * [见微知著：巴斯德所开发深度学习技术，极大提速超分辨率显微成像](https://ibook.antpedia.com/x/288405.html), BioArt (2019, news)