<!DOCTYPE html>
<html lang="zh">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="renderer" content="webkit">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title>
        Wei OUYANG
    </title>
    
<link rel="stylesheet" href="/libs/highlight/styles/monokai-sublime.css">

    
<link rel="stylesheet" href="/libs/font-awesome/css/font-awesome.min.css">

    
<link rel="stylesheet" href="/css/style.css">

<meta name="generator" content="Hexo 5.4.0"></head>

<body id="bodyx">
    <div class="hd logodiv">
    <!-- https://getavataaars.com/?accessoriesType=Prescription02&avatarStyle=Transparent&clotheColor=Gray01&clotheType=Hoodie&eyeType=Happy&eyebrowType=DefaultNatural&facialHairType=Blank&hairColor=Black&mouthType=Twinkle&skinColor=Pale&topType=ShortHairShortFlat -->
    <img src="/img/wei-avatar.svg" class="logopng">
    <p class="logot"><span class="logo_1">&lt;</span><span class="logo_2">/</span><span class="logo_3">#</span> Wei OUYANG <span class="logo_3">#</span><span class="logo_2">/</span><span class="logo_1">&gt;</span></p>
</div>
<div class="hd">
    <ol class="breadcrumb">
        
            
                <li><a class="ba" href="/about">About Me</a></li>
            
        
            
                <li><a class="ba bold" href="/publications">Publications</a></li>
            
        
            
                <li><a class="ba" href="/projects">Projects</a></li>
            
        
            
                <li><a class="ba" target="_blank" rel="noopener" href="https://github.com/oeway">Github</a></li>
            
        
    </ol>
    <hr class="breadcrumbhr">
</div>
<div class="hd posts">
    <!-- <a href="/index.html"><i class="fa fa-reply replay-btn" aria-hidden="true"></i></a> -->
    
        <br>
    
    <div class="post-content">
        <h2 id="Papers"><a href="#Papers" class="headerlink" title="Papers"></a>Papers</h2><!-- * **W. Ouyang**, Emma Lundberg, ImageJ.JS: ImageJ running in the browser, https://ij.imjoy.io/ (2021) -->
<!-- * J. Bai<sup>*</sup>, **W. Ouyang<sup>*#</sup>**, M. Singh, C. Leterrier, P. Barthelemy, S. Holden, T. Klein, M. Sauer, P. Kanchanawong, N. Bourg, M. Cohen, B. Lelandais, C. Zimmer<sup>#</sup>, ShareLoc – an open platform for sharing localization microscopy data (2021) -->
<!-- * M.R. King, K.M. Ruff, X. Zeng, A.E. Posey, W. Ouyang, E. Lundberg, R.V. Pappu. Nucleolar cores are scaffolded by proteins with lysine and acid-rich disordered regions that drive phase separation. In preparation. -->
<ul>
<li><p>E. Gómez-de-Mariscal, C. García-López-de-Haro, <strong>W. Ouyang</strong>, L. Donati, E. Lundberg, M. Unser, A. Muñoz-Barrutia, D. Sage. DeepImageJ: A user-friendly environment to run deep learning models in ImageJ, Nature Methods (2021, in press), <a target="_blank" rel="noopener" href="https://www.biorxiv.org/content/10.1101/799270v3">preprint</a></p>
</li>
<li><p>A. Imbert, <strong>W. Ouyang</strong>,  A. Safieddine, E. Coleno, C. Zimmer, E. Bertrand, T. Walter, F. Mueller. FISH-quant v2: a scalable and modular analysis tool for smFISH image analysis, Biorxiv (2021, submitted to Nucleic Acids Research). <a target="_blank" rel="noopener" href="https://doi.org/10.1101/2021.07.20.453024">https://doi.org/10.1101/2021.07.20.453024</a></p>
</li>
<li><p>X. Hao, J.J. Parmar, B. Lelandais, A. Aristov, <strong>W. Ouyang</strong>, C. Weber &amp; C. Zimmer. Super-resolution visualization and modeling of human chromosomal regions reveals cohesin-dependent loop structures. Genome Biology (2021). <a target="_blank" rel="noopener" href="https://doi.org/10.1186/s13059-021-02343-w">https://doi.org/10.1186/s13059-021-02343-w</a></p>
</li>
<li><p><strong>W. Ouyang</strong>, R. Bowman, H. Wang, K. E Bumke, J. T Collins, O. Spjuth, J. Carreras-Puigvert, B. Diederich, An Open-Source Modular Framework for Automated Pipetting and Imaging Applications, Biorxiv (2021, submitted to Advanced Biology).<a target="_blank" rel="noopener" href="https://doi.org/10.1101/2021.06.24.449732">https://doi.org/10.1101/2021.06.24.449732</a></p>
</li>
<li><p>H. Pinkard, N. Stuurman, I.E. Ivanov, <strong>W. Ouyang</strong> et al., Pycro-Manager: open-source software for customized and reproducible microscope control. Nature Methods (2021). <a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41592-021-01087-6">https://doi.org/10.1038/s41592-021-01087-6</a>, <a target="_blank" rel="noopener" href="https://rdcu.be/cghwk">full text</a></p>
</li>
<li><p><strong>W. Ouyang</strong>, T. Le, H. Xu and E. Lundberg. Interactive biomedical segmentation tool powered by deep learning and ImJoy. F1000Research (2021). <a target="_blank" rel="noopener" href="https://doi.org/10.12688/f1000research.50798.1">https://doi.org/10.12688/f1000research.50798.1</a></p>
</li>
<li><p>Y. Qin, C. F. Winsnes, E. L. Huttlin, F. Zheng, <strong>W. Ouyang</strong>, J. Park, A. Pitea, J. F. Kreisberg, S. P. Gygi, J. W. Harper, J. Ma, E. Lundberg, T. Ideker, Mapping cell structure across scales by fusing protein images and interactions, Nature (2021, in press). <a target="_blank" rel="noopener" href="https://doi.org/10.1101/2020.06.21.163709">preprint</a></p>
</li>
<li><p>L. Stenström, D. Mahdessian, C. Gnann, A. J Cesnik, <strong>W. Ouyang</strong>, M. D. Leonetti, M. Uhlén, S. Cuylen‐Haering, P. J Thul, E. Lundberg, Mapping the nucleolar proteome reveals a spatiotemporal organization related to intrinsic protein disorder. Mol Syst Biol. (2020). <a target="_blank" rel="noopener" href="https://doi.org/10.15252/msb.20209469">https://doi.org/10.15252/msb.20209469</a></p>
</li>
<li><p><strong>W. Ouyang</strong>, F. Mueller, M. Hjelmare, E. Lundberg, C. Zimmer, ImJoy: an open-source computational platform for the deep learning era, Nature Methods, 2019, <a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41592-019-0627-0">https://doi.org/10.1038/s41592-019-0627-0</a>, <a target="_blank" rel="noopener" href="https://rdcu.be/bYbGO">full text</a></p>
</li>
<li><p><strong>W. Ouyang</strong> et. al, Analysis of the Human Protein Atlas Image Classification competition, Nature Methods, 2019, <a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41592-019-0658-6">https://doi.org/10.1038/s41592-019-0658-6</a>, <a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41592-019-0658-6">full text</a></p>
</li>
<li><p><strong>W. Ouyang</strong>, A. Aristov, M. Lelek, X. Hao, C. Zimmer*, Deep learning massively accelerates super-resolution localization microscopy, Nature Biotechnology, 2018, <a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.4106">https://doi.org/10.1038/nbt.4106</a>, <a target="_blank" rel="noopener" href="https://rdcu.be/LGtc">full text</a>, <a target="_blank" rel="noopener" href="https://www.nature.com/nbt/volumes/36/issues/5">cover design</a></p>
</li>
<li><p>A. Samacoits, R. Chouaib, A. Safieddine, A. Traboulsi, <strong>W. Ouyang</strong>, C. Zimmer, M. Peter, E. Bertrand, T. Walter, F. Mueller, A computational framework to study sub-cellular mRNA localization, Nature Communications, 2018, <a target="_blank" rel="noopener" href="https://doi.org/10.1038/s41467-018-06868-w">https://doi.org/10.1038/s41467-018-06868-w</a></p>
</li>
<li><p>G. Yang et al., IoT-based Remote Pain Monitoring System: from Device to Cloud Platform, IEEE Journal of Biomedical and Health Informatics, 2017, <a target="_blank" rel="noopener" href="https://doi.org/10.1109/JBHI.2017.2776351">https://doi.org/10.1109/JBHI.2017.2776351</a></p>
</li>
</ul>
<h2 id="Review"><a href="#Review" class="headerlink" title="Review"></a>Review</h2><ul>
<li><strong>W. Ouyang</strong>, C. Zimmer*, The imaging tsunami: computational opportunities and challenges, Current Opinion in Systems Biology, 2017, <a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.coisb.2017.07.011">https://doi.org/10.1016/j.coisb.2017.07.011</a></li>
</ul>
<h2 id="News-and-Highlights"><a href="#News-and-Highlights" class="headerlink" title="News and Highlights"></a>News and Highlights</h2><ul>
<li><a target="_blank" rel="noopener" href="https://www.scilifelab.se/news/ai-is-changing-the-future-of-life-science-but-how/">AI is changing the future of life science – but how?</a>, SciLifeLab(2021, interview)</li>
<li><a target="_blank" rel="noopener" href="https://www.nature.com/articles/d41586-021-00023-0">Sharper signals: how machine learning is cleaning up microscopy images</a>, Nature (2021, technology features)</li>
<li><a target="_blank" rel="noopener" href="https://www.nature.com/articles/d41586-020-01128-8">Deep learning takes on tumours</a>, Nature (2020, technology features)</li>
<li><a target="_blank" rel="noopener" href="https://www.scilifelab.se/news/imjoy-a-new-computational-platform-for-deep-learning/">ImJoy: a new computational platform for deep learning</a>, SciLifeLab(2020, news)</li>
<li><a target="_blank" rel="noopener" href="https://www.pasteur.fr/en/research-journal/news/artificial-intelligence-new-platform-biomedical-community">ARTIFICIAL INTELLIGENCE: A NEW PLATFORM FOR THE BIOMEDICAL COMMUNITY</a>, Institut Pasteur (2020, news)</li>
<li><a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41592-018-0028-9">Deep learning advances super-resolution imaging</a>, Nature Methods (2018, Research Highlight)</li>
<li><a target="_blank" rel="noopener" href="https://physicsworld.com/a/super-resolution-microscopy-gets-a-dose-of-deep-learning/">Super resolution microscopy gets a dose of deep learning</a>, Physics World (2018)</li>
<li><a target="_blank" rel="noopener" href="https://www.news-medical.net/life-sciences/Applying-Deep-Learning-to-Localization-Microscopy-(Portuguese).aspx">Aplicando profundamente a aprendizagem à microscopia da localização</a>, Medical News (2020)</li>
<li><a target="_blank" rel="noopener" href="https://www.inception-program.fr/en/news/artificial-intelligence-speeds-high-resolution-microscopy">ARTIFICIAL INTELLIGENCE SPEEDS UP HIGH-RESOLUTION MICROSCOPY</a>, Institut Pasteur (2018, news)</li>
<li><a target="_blank" rel="noopener" href="https://ibook.antpedia.com/x/288405.html">见微知著：巴斯德所开发深度学习技术，极大提速超分辨率显微成像</a>, BioArt (2019, news)</li>
</ul>

    </div>
    
        <hr class="fhr">
        <div id="vcomments"></div>
    
</div>
    <div class="footer" id="footer">
    <p>Copyright © 2020 <a class="flink" href="#">Wei OUYANG</a>.
        <label class="el-switch el-switch-green el-switch-sm" style="vertical-align: sub;">
            <input type="checkbox" name="switch" id="update_style">
            <span class="el-switch-style"></span>
        </label>
<!--         <script type="text/javascript">
        var cnzz_protocol = (("https:" == document.location.protocol) ? "https://" : "http://");
        document.write(unescape("%3Cspan id='cnzz_stat_icon_1278548644'%3E%3C/span%3E%3Cscript src='" + cnzz_protocol + "v1.cnzz.com/stat.php%3Fid%3D1278548644%26show%3Dpic1' type='text/javascript'%3E%3C/script%3E"));
        </script> -->
    </p>
</div>
<input type="hidden" id="web_style" value="black">
<input type="hidden" id="valine_appid" value="">
<input type="hidden" id="valine_appKey" value="">

<script src="/libs/jquery.min.js"></script>


<script src="/libs/highlight/highlight.pack.js"></script>

<script src='//cdn.jsdelivr.net/npm/valine@1.3.10/dist/Valine.min.js'></script>

<script src="/js/js.js"></script>

<style type="text/css">
.v * {
    color: #698fca;
}

.v .vlist .vcard .vhead .vsys {
    color: #3a3e4a;
}

.v .vlist .vcard .vh .vmeta .vat {
    color: #638fd5;
}

.v .vlist .vcard .vhead .vnick {
    color: #6ba1ff;
}

.v a {
    color: #8696b1;
}

.v .vlist .vcard .vhead .vnick:hover {
    color: #669bfc;
}
</style>
</body>

</html>